capstone/backend/src/main/java/com/google/servlets/StoreRankingsServlet.java

```diff
@@ -0,0 +1,179 @@
+/*
+ * Copyright 2020 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.servlets;
+
+import com.google.api.client.http.GenericUrl;
+import com.google.api.client.http.HttpRequest;
+import com.google.api.client.http.HttpRequestFactory;
+import com.google.api.client.http.javanet.NetHttpTransport;
+import com.google.cloud.spanner.SpannerException;
+import com.google.gson.Gson;
+import com.google.spanner.LibraryFunctions;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/api/v1/get-store-rankings")
+public class StoreRankingsServlet extends HttpServlet {
+  private static final String API_KEY = "INSERT_GOOGLE_API_KEY_HERE";
+  private static final double AVALIABLE_ITEMS_WEIGHT = 3;
+  private static final double DISTANCE_WEIGHT = -0.0005;
```

Comment:
Add some comments to these weight constants?

Reply:
Once I talk to Brett and Carolyn and we finalize the algorithm and weights we are going to use, I will link a document explaining how the weights work.